code/game/objects/items/weapons/storage/bags.dm

```diff
@@ -375,7 +375,7 @@
 	max_combined_w_class = 21
 	max_w_class = 3
 	w_class = 4 //Bigger than a book because physics
-	can_hold = list("/obj/item/weapon/book", "/obj/item/weapon/spellbook") //No bibles, consistent with bookcase
+	can_hold = list("/obj/item/weapon/book", "/obj/item/weapon/storage/bible", "/obj/item/weapon/tome", "/obj/item/weapon/spellbook")

```

Comment:
Why are these text paths?
Why did somebody think [this](https://github.com/ParadiseSS13/Paradise/blob/master/code/game/objects/items/weapons/storage/storage.dm#L247) was a good idea?
WHY OLDCODERS WHY?!